Documentation/yugabytedb-cluster-crd.md

```diff
@@ -78,6 +94,11 @@ Specify Master/TServer specific attributes under `master`/`tserver`. The valid a
 
 Specify replica count for `master` & `tserver` pods under `replicas` field. This is a **required** field.
 
+### Resource
+
+Specify resource requests and limits for CPU & Memory. If provided, the given resource values will be used. If omitted the default CPU request & limit will be 2, whereas default memory request & limit will be 2Gi & 4Gi for Master & TServer, respectively. You may override these values for dev environments where the cluster doesn't have CPU/Memory as per the defaults.
```

Comment:
Production scenarios all have the same requirements? It doesn't depend on the workload?